config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "control_channels": [1069014613779300524]
```

Comment:
Is this the control channel in the 110 server?

Reply:
Yes